src/app/core/models/parser/csvData.model.ts

```diff
@@ -0,0 +1,13 @@
+import { Admins } from "./admins.model";
+import { Roles } from "./roles.model";
+import { Students } from "./students.model";
+import { Teams } from "./teams.model";
+import { Tutors } from "./tutors.model";
+
+export interface CSVData {
```

Comment:
`CSVData` sounds a bit too vague to be used in this context. 

Was thinking of adopting another name for this type. 

Do you have any suggestions? @CATcher-org/2021-devs @zatkiller

Reply:
I put it as CSVdata because I wasn't quite sure how to name it either, since its made up of many different components. Since this data type consists of many different types of user data, would something like `tabulatedUserData` be better?

Reply:
Sounds good! do remember to rename the file to `tabulated-user-data.model.ts` as well! :D